packages/design-system-docs/src/pages/utilities/layout-grid/_grid.docs.scss

```diff
@@ -12,7 +12,7 @@ The grid consists of three distinct pieces:
 
 Flexbox grid supports modern browsers and Internet Explorer 10+. In older versions of Internet Explorer the grid columns will be stacked.
 
-Style guide: styles.grid
+Style guide: utilities.grid
```

Comment:
Similar to USWDS, we want to ensure that Layout Grid appears first. I'm not sure if this is currently possible with KSS, so you might want to add that capability by adding this code to `processKssSection`

```
case 'weight':
            section.weight = value;
            break;
```

Reply:
`processKssSection` doesn't have the `weight` property. I am checking the `section.referenceURI` to update the `section.weight` instead. Think that's workable @bernardwang ?

Reply:
I was suggesting you add the `weight` property with that code i shared above.